Add initial German translation

vmx opened the pull request:
@rixx if you tell me how to properly add a the formal German translation file, I can also add that.

@astroidex Can you please have a look at the translation (or ping someone else to have a look)?

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -19,15 +19,15 @@ msgstr ""
 #: pretalx_public_voting/exporters.py:16
 msgid "Public Voting CSV"
-msgstr ""
+msgstr "CSV der öffentliche Abstimmung"
```
der öffentlichen - ein e fehlt

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 "\n"
 "The {event.name} organisers\n"
```
"Das {event.name} Organisationsteam"

vmx replied:
Accendental comment, this one is the english one.

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 msgstr ""
+"Hi,\n"
+"\n"
+"du hast dich registriert um über Einreichungen von {event.name} abzustimmen\n"
```
"du hast dich registriert, um über Einreichungen zur {event.name} abzustimmen.\n"

vmx replied:
Wie wäre es mit "du hast dich bei {event.name} registriert um über die Einreichungen abzustimmen"? Das ist jetzt keine grammatikalische Hochleistung, allerdings funktioniert es halbwegs sowohl mit "FOSSGIS" als auch "Starkbieranstich".

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
+"\n"
+"    {vote_url}\n"
+"\n"
+"Falls du nicht dich nicht für die Stimmabgabe registriert hast, ignorier "
```
"Falls du dich nicht für die Stimmabgabe registriert hast, kannst du  "

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
+"    {vote_url}\n"
+"\n"
+"Falls du nicht dich nicht für die Stimmabgabe registriert hast, ignorier "
+"diese E-Mail einfach.\n"
```
"diese E-Mail einfach ignorieren.\n"

vmx replied:
Die Idee kam von https://translate.pretalx.com/translate/pretalx/pretalx/de/?checksum=8c269ce491eeedc3, aber ich finde deine Übersetzung besser (werde das also so machen), vielleicht sollte das an der anderen Stelle auch weniger flapsig formuliert werden.

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
+"Falls du nicht dich nicht für die Stimmabgabe registriert hast, ignorier "
+"diese E-Mail einfach.\n"
+"\n"
+"Vielen Dank, dass du bei der Abstimmung teilnimmst!\n"
```
"Vielen Dank, dass du an der Abstimmung teilnimmst!\n"

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 "publicly visible."
 msgstr ""
+"Nach diesem Zeitpunkt ist keine öffentliche Abstimmung mehr möglich. Die "
+"Einreichungen sind dann icht mehr öffentlich einsehbar."
```
Einreichungen sind dann nicht mehr öffentlich einsehbar."

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 #: pretalx_public_voting/forms.py:124
 msgid "Hide speaker names and use anonymized content where available?"
-msgstr ""
+msgstr "Wenn möglich, Namen der Vortrageden und anonymisierten Text verwenden?"
```
Vortragenden

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 #: pretalx_public_voting/forms.py:134
 msgid "The minimum score voters can assign"
-msgstr ""
+msgstr "Die geringste Wertung. die vergeben werden kann."
```
, statt .

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 msgstr ""
+"Die öffentliche Abstimmung zeigt alle Einreichungen öffentlich an und "
+"ermöglicht es jedem, der eine gültige E-Mail-Adresse hat, abzustimmen. Die "
+"E-Mail-Adressen werden nicht gespeichert, es ist also anonym."
```
"E-Mail-Adressen werden nicht gespeichert. Die Abstimmung ist also anonym."

astroidex commented on pretalx_public_voting/locale/de_DE/LC_MESSAGES/django.po:
```diff
@@ -45,161 +45,195 @@ msgid ""
 #: pretalx_public_voting/templates/pretalx_public_voting/submission_list.html:59
 msgid "Click here to sign up for voting."
-msgstr ""
+msgstr "Bitte klicke hier um dich für das Bewerten anzumelden."
```
, vor um

rixx commented:
@vmx ``makemessages`` should create a ``de_Formal`` directory for you, or you could just copy the ``de_DE`` one. You can compare with pretalx core to see what it should look like.

Please let me know when you want this PR merged.

vmx commented:
@astroidex I've updated the translation.

I took your review (thanks!) and took all the changes, except for one where I used what I propose [in this comment](https://github.com/pretalx/pretalx-public-voting/pull/17#discussion_r571687605). The full changes can be found here: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/e152167a43d31123e2a75ec7e722134b58da38e7

I also found more typos/bugs. Those changes can be found here: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/248b2078a86cea1217b21340840d863788102f8b

And finally I've added a formal translation (is that one used for the FOSSGIS?). In order to make the review easier, here the diff which only shows the difference between the formal and non-formal version: https://github.com/pretalx/pretalx-public-voting/pull/17/commits/502713951f9ab2db2ce0b3daead84fb10d3c3969

astroidex commented:
fixed some typos. I like the formal version for FOSSGIS

vmx commented:
@rixx This is ready to be merged. If it needs a rebase, let me know and I'll do that.

When this one is merged, the current state is good enough for the FOSSGIS. Could you please then deploy this version?

rixx commented:
> When this one is merged, the current state is good enough for the FOSSGIS. Could you please then deploy this version?

I've been continually deploying after every PR merge ;)

vmx commented:
Looks like we broke something. I get a 500 on signup :-/ (i can have a look later tonight if it also happens locally)

rixx commented:
No, that was me trying to fix the misformatted email. Fixed now.
